right/src/config_parser/parse_config.c

```diff
@@ -118,6 +104,40 @@ parser_error_t ParseConfig(config_buffer_t *buffer)
         return ParserError_InvalidMouseKineticProperty;
     }
 
+    // Version 7:
+
+    float mouseMoveAxisSkew = 1.0f;
+    float mouseScrollAxisSkew = 1.0f;
+    bool diagonalSpeedCompensation = false;
+
+    uint16_t doubletapTimeout = DoubletapTimeout;
+    uint16_t keystrokeDelay = KeystrokeDelay;
+
+    secondary_role_strategy_t secondaryRoles_Strategy = SecondaryRoleStrategy_Simple;
+    uint16_t secondaryRoles_AdvancedStrategyDoubletapTime = SecondaryRoles_AdvancedStrategyDoubletapTime;
+    uint16_t secondaryRoles_AdvancedStrategyTimeout = SecondaryRoles_AdvancedStrategyTimeout;
+    int16_t secondaryRoles_AdvancedStrategySafetyMargin = SecondaryRoles_AdvancedStrategySafetyMargin;
+    bool secondaryRoles_AdvancedStrategyTriggerByRelease = SecondaryRoles_AdvancedStrategyTriggerByRelease;
+    bool secondaryRoles_AdvancedStrategyDoubletapToPrimary = SecondaryRoles_AdvancedStrategyDoubletapToPrimary;
+    serialized_secondary_role_action_type_t secondaryRoles_AdvancedStrategyTimeoutAction = SerializedSecondaryRoleActionType_Secondary;
+
+    if (DataModelMajorVersion >= 7) {
+        secondaryRoles_Strategy = ReadUInt8(buffer);
+        secondaryRoles_AdvancedStrategyDoubletapTime = ReadUInt16(buffer);
```

Comment:
should not it be `secondaryRoles_AdvancedStrategyDoubletapTimeout` ? 
There is timeout on the Agent UI and timeout in the ["documentation"](https://github.com/UltimateHackingKeyboard/agent/issues/1921#issuecomment-1913575443)

Reply:
Yes.